projects/route-analyzer/src/cli.ts

```diff
@@ -17,26 +19,43 @@ const argv = yargs
     })
     .option('outputFile', {
         alias: 'o',
-        description: 'Entry file of your angular application',
+        description: 'JSON routes file to be generated by this script',
+        deprecated: 'Use outputJson instead',
+        type: 'string',
+    })
+    .option('outputJson', {
```

Comment:
What about the idea that you had of specifying the output directory?
It is a little bit awkward to specify an output for file type - json & typescript

Reply:
If we're OK with breaking backwards compatibility, I could do the directory, but that forces us to hard code a name. @ryan-bradford What do you think?

Reply:
I think output directory might be nice. Can we just support both? Like four options:
1. Specify output directory
2. Manually specify both filed
3. Specify just JSON
4. Specify just TS